internals/webpack/webpack.base.babel.js

```diff
@@ -1,10 +1,34 @@
 /**
  * COMMON WEBPACK CONFIGURATION
  */
+require('dotenv').config();
 
 const path = require('path');
 const webpack = require('webpack');
 
+const rawEnvVars = Object
+  .keys(process.env)
+  .filter((key) => /^RB_/i.test(key))
+  .reduce((env, key) => {
+    env[key] = process.env[key]; // eslint-disable-line no-param-reassign
+    return env;
+  }, {
+    // Always expose NODE_ENV to webpack, in order to use `process.env.NODE_ENV`
+    // inside your code for any environment checks; UglifyJS will automatically
+    // drop any unreachable code.
+    NODE_ENV: JSON.stringify(process.env.NODE_ENV),
+  })
+;
+
+const StringifiedEnvVars = {
+  'process.env': Object
+    .keys(rawEnvVars)
+    .reduce((env, key) => {
```

Comment:
To avoid disabling eslint you can use this approach:

```javascript
...
.reduce((env, key) => ({ ...env, [key]: JSON.stringify(rawEnvVars[key] }), {})
... 

```